srcpkgs/python3-poetry-core/template

```diff
@@ -1,24 +1,20 @@
 # Template file for 'python3-poetry-core'
 pkgname=python3-poetry-core
-version=1.5.0
-revision=2
+version=1.9.0
+revision=1
 build_style="python3-pep517"
 depends="python3"
 checkdepends="python3-devel python3-setuptools python3-virtualenv python3-pyrsistent
- python3-pytest-mock"
+ python3-pytest-mock python3-tomli-w"
 short_desc="Poetry PEP 517 Build Backend & Core Utilities"
-maintainer="Kye Shi <shi.kye@gmail.com>"
+maintainer="Leon (adigitoleo) <adigitoleo@posteo.net>"
```

Comment:
We like to see a longer history of contributions before packages are adopted. Please leave the maintainer field as-is for now.

Reply:
when noticed, long inactive maintainers get their packages orphaned

Reply:
https://github.com/void-linux/void-packages/commit/872446166b3cc602c1a9cbc4cae4a8f1dbf1b518